### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by WongIong** [SawFowl/CommandSync#26]
Is that possible to have a Bungee using CSC and make it connect to CSS?
Because i have two Proxies now and if i want to do sth i have to do that twice

**Comment by SawFowl** [comment]
No, you won't be able to configure the plugin that way. I do not think that in the foreseeable future I will implement the plugin on multiple proxy servers simultaneously.

**Comment by WongIong** [comment]
is that not able to do in theory?

**Comment by SawFowl** [comment]
This can be done by augmenting the plugin code. The easiest way to implement such a feature would be to augment the functionality of the plugin on proxy servers with client functions. In this case, the plugin will work as a client. But right now I do not have time to implement it. I'm busy with another, more complicated project and there are a few more projects in the queue that I will need to open my own server in the future.

**Comment by WongIong** [comment]
oh get it, its ok i could wait :)
